### PR TITLE
Remove all console errors

### LINKED ISSUE #232: Console Errors
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
After the recent updates in chromebot there are console error persisting in it.

**Expected behavior:**
There should be no console errors in it

**Steps to reproduce:**
Open Developer console in chromebot

**Screenshot**
screenshot from 2017-12-18 09-22-47

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by ms10398**
Fixes #232 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Removed all console errors

#### Changes proposed in this pull request:
Removed all console errors

### CODE REVIEW

**Comment by ms10398** [comment]
@gabru-md @stealthanthrax @zamhaq Please review

File: src/script.js [hunk 1]
```diff
@@ -27,13 +27,16 @@ var backUrl = localStorage.getItem("theValue");
...
-    headerbox.style.backgroundColor = theme;
-    mic.style.color = theme;
-    but.style.color = theme;
-    console.log(theme);
+
+    if(backUrl) {
+        box.style.backgroundImage = "url(" + backUrl + ")";
+        box.style.backgroundRepeat = "no-repeat";
+        box.style.backgroundSize = "cover";
+        headerbox.style.backgroundColor = theme;
```
Comment (stealthanthrax) [inline src/script.js]: @ms10398 ,shouldn't lines 35,36,37,38  be in a different check?

Comment (ms10398) [reply src/script.js]: I didnt got it?
What do you mean exactly

**Comment by ms10398** [comment]
Check it now

**Comment by gabru-md** [comment]
@zamhaq  please merge

**Pull request merged**